tests/04_00_02-fedchange.js

```diff
@@ -54,33 +57,44 @@ const PENDING_FEDERATION_RANDOM_HASH = '0x27da3e662acf8862ba5c42fbdcc023c4f81355
 const INITIAL_BTC_BALANCE = bitcoin.btcToSatoshis(200);
 const EXPECTED_UNSUCCESSFUL_RESULT = -10;
 
-var btcClient;
-var rskClientOldFed;
-var rskClientNewFed;
-var rskClients;
-var pegClient;
-var test;
-var testNewFed;
-var utils;
-var expectedNewFederationCreationTime, expectedNewFederationCreationBlockNumber;
-var oldFederation;
-var newFederationBtcPublicKeys;
-var newFederationPublicKeys;
-var newFederatorRskAddressesRsk;
-var expectedNewFederationAddress;
+let btcClient;
+let rskClientOldFed;
+let rskClientNewFed;
+let rskClients;
+let pegClient;
+let test;
+let testNewFed;
+let utils;
+let expectedNewFederationCreationTime, expectedNewFederationCreationBlockNumber;
+let oldFederation;
+let newFederationBtcPublicKeys;
+let newFederationPublicKeys;
+let newFederatorRskAddressesRsk;
+let expectedNewFederationAddress;
 let p2shErpFedRedeemScript;
-var expectedNewFederationThreshold;
+let expectedNewFederationThreshold;
 let amountOfUtxosToMigrate;
 let federationBalanceBeforeMigration;
 let whitelistingAssertionsTestLegacy;
 let rskTxHelpers;
 let btcTxHelper;
+let rskTxHelper;
+
+/**
+ * Takes the blockchain to the required state for this test file to run in isolation.
+ */
+const fulfillRequirementsToRunAsSingleTestFile = async (rskTxHelper, btcTxHelper) => {
+  await rskUtils.activateFork(Runners.common.forks.fingerroot500);
```

Comment:
I'd change this activate the latest fork, if not we'll have to change this every time a new network upgrade is released

Reply:
Updated!